src/main/resources/db/migration/V7__EventToSendInsertionTime.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE event_to_send ADD column insertion_time TIMESTAMP NOT NULL DEFAULT '2021-01-01 00:00:00.000';
```

Comment:
I know that the JIRA loosely described an "insertion time" , but we are now heading towards a wide variety of names for the creation timestamp. 
Other tables have `created_at` (from RM) , or newer ones just `date_time` . 
I would like us to decide on a common naming convention

Reply:
The insertion_time (or whatever it ends up being called) needs to be indexed

Reply:
Done